packages/db/src/migrate.ts

```diff
@@ -1,3 +1,4 @@
+import "dotenv/config";
```

Comment:
We shouldn't need this. The scripts in `package.json` files wrap all code execution in `dotenv`, so the code should not need to do anything to set up environment variables.